_maps/RandomRuins/IceRuins/icemoon_underground_alien_lab.dmm

```diff
@@ -0,0 +1,1325 @@
+/area/ruin/powered/alien_lab)
+"fW" = (
+/turf/template_noop,
+/area/template_noop)
+"gc" = (
+/mob/living/carbon/human/moja,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"iW" = (
+/obj/structure/flora/tree/dead,
+/turf/open/floor/grass/snow,
+/area/ruin/powered/alien_lab)
+"kT" = (
+/obj/effect/decal/cleanable/blood/gibs/old,
+/obj/effect/decal/cleanable/blood/gibs/down,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"ln" = (
+/obj/effect/decal/cleanable/blood/drip,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"lN" = (
+/obj/effect/decal/cleanable/blood/drip,
+/turf/open/floor/grass/snow,
+/area/ruin/powered/alien_lab)
+"mS" = (
+/obj/effect/decal/cleanable/blood/gibs/up,
+/obj/structure/bed/abductor,
+/mob/living/carbon/human/moja,
+/obj/item/organ/heart/gland/heals,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"nJ" = (
+/obj/effect/decal/cleanable/blood/tracks,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"rg" = (
+/obj/structure/flora/tree/pine,
+/turf/open/floor/grass/snow,
+/area/ruin/powered/alien_lab)
+"rY" = (
+/obj/machinery/light/small,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"uW" = (
+/obj/effect/decal/cleanable/blood/gibs/body{
+	dir = 1
+	},
+/obj/effect/decal/cleanable/blood/gibs/limb{
+	dir = 1
+	},
+/obj/machinery/light/broken{
+	dir = 1
+	},
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"vi" = (
+/obj/effect/decal/cleanable/blood/gibs/down,
+/obj/effect/decal/cleanable/blood/gibs/down,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"wa" = (
+/obj/structure/door_assembly/door_assembly_abductor,
+/turf/open/floor/grass/snow,
+/area/ruin/powered/alien_lab)
+"wD" = (
+/mob/living/simple_animal/hostile/abomination/altform2,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"wQ" = (
+/obj/effect/decal/cleanable/blood/gibs,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"xi" = (
+/obj/effect/decal/cleanable/blood/gibs/torso,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"xI" = (
+/mob/living/simple_animal/hostile/abomination,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"yj" = (
+/mob/living/carbon/human/moja,
+/turf/open/lava/plasma,
+/area/ruin/powered/alien_lab)
+"yO" = (
+/mob/living/simple_animal/hostile/asteroid/wolf,
+/obj/item/reagent_containers/food/snacks/meat/slab/human,
+/turf/open/indestructible/grass/snow,
+/area/ruin/powered/alien_lab)
+"Af" = (
+/obj/structure/table/abductor,
+/obj/item/abductor/baton,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"AM" = (
+/mob/living/simple_animal/hostile/skeleton/plasmaminer/jackhammer,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"AR" = (
+/obj/machinery/light/broken,
+/turf/open/floor/plating/abductor,
+/area/ruin/powered/alien_lab)
+"AZ" = (
+/obj/structure/grille,
```

Comment:
```suggestion

```
spawners already put down window + grille, this would be putting down 1 extra grille